### DISCUSSION TITLE
[Requested Feature] Circular Graph Capabilities

### DISCUSSION THREAD

**Discussion by andrewtran249** [MicroHealthLLC/mSuite#627] [Ideas]
Ability to create circular graphs, venn diagrams, pie charts

**Comment by bluenevus** [comment]
there are requirements for this already

https://github.com/MicroHealthLLC/mSuite/issues/142
https://github.com/MicroHealthLLC/mSuite/issues/137
https://github.com/MicroHealthLLC/mSuite/issues/136
